### PR TITLE
Add H3 layer

### PR DESCRIPTION
**Pull request by ericsoco**
Still planning to add a coverage slider to match existing Hexbin Layer.
Also, I think @nrabinowitz has some additional edits.

This addresses #154, but does not add support for H3 bin aggregation (that can come in a followup diff, extending our use of `h3-js`.

screen shot 2018-08-14 at 11 20 27 am

### CODE REVIEW

**Comment by heshan0131** [comment]
This is awesome. Thank you for doing this. Did you moved the files from @uber/kepler.gl?

File: yarn.lock [hunk 1]
```diff
@@ -3506,6 +3506,10 @@ grid-index@^1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/grid-index/-/grid-index-1.0.0.tgz#ad2c5d54ce5b35437faff1d70a9aeb3d1d261110"
 
+h3-js@^3.1.0:
```
Comment (heshan0131) [inline yarn.lock]: Please update your `~/.npmrc` to use `registry = https://registry.yarnpkg.com/` before running yarn. This is why the build failed

**Comment by ericsoco** [comment]
@heshan0131 yep, copied from internal kepler wrapper. Will try to find time this week to wrap it up, TBD.

**Comment by ericsoco** [comment]
@ibgreen perhaps? Thoughts on where in deck.gl this would be added, `experimental-layers` to start I assume? (Been a while since I contributed.)

Would probably want to standardize `enhanced-hexagon-cell-layer` across this PR and the [existing one](https://github.com/uber/kepler.gl/blob/master/src/deckgl-layers/hexagon-layer/enhanced-hexagon-cell-layer.js) in Kepler, and may need some other cleanup.

I'm guessing the best path here is to land this into Kepler first and then work w/ @heshan0131 and @nrabinowitz to tidy things up enough to hoist them into deck.gl. Thoughts?

**Comment by ibgreen** [comment]
@ericsoco - All sounds reasonable.

Also see the ideas in the [deck.gl layers roadmap](https://github.com/uber/deck.gl/blob/master/dev-docs/roadmaps/layers-roadmap.md) where we start discussing having multiple layer catalogs.

Putting things in experimental layers is a nice first start, since more users can access the layers.

The big job is typically getting layers out of experimental layers and into the official catalog(s).

**Comment by heshan0131** [comment]
I can take over this PR. Please grant me access to your fork. You can [allow edits from maintainers](
https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/)

**Pull request merged**